.github/workflows/hvd-tests.yml

```
@@ -78,7 +78,7 @@ jobs:
           bash tests/run_cpu_tests.sh
 
       - name: Upload coverage to Codecov
-        uses: codecov/codecov-action@v1
+        uses: codecov/codecov-action@v3.1.1
```

Here and elsewhere let's only keep major version:
```suggestion
        uses: codecov/codecov-action@v3
```